submissions/webdevtoday/oop_exercise/index.js

```diff
@@ -0,0 +1,76 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammal {
+   legs = 4;
+
+   constructor(name, gender, friends) {
+      this.name = name;
+      this.gender = gender;
+      this.friends = friends;
+   }
+}
+
+class Human extends Mammal {
+   legs = 2;
```

Comment:
`legs` prop already defined by a superclass. It's an antipattern to redefine an inherited property. It should be handled by property owning class.

Reply:
I don't get you(( Do I need to create getter and setter for that property?